RecoLocalTracker/SiPixelRecHits/plugins/SiPixelRecHitProducer.cc

```diff
@@ -0,0 +1,137 @@
+#include "DataFormats/Common/interface/Handle.h"
```

Comment:
A (failed?) attempt to consolidate producers. Be either removed or eventually make it useful.